libraries/SPISlave/src/hspi_slave.c

```diff
@@ -92,7 +92,7 @@ void hspi_slave_begin(uint8_t status_len, void * arg)
     SPI1S1 = (((status_len * 8) - 1) << SPIS1LSTA) | (0xff << SPIS1LBUF) | (7 << SPIS1LWBA) | (7 << SPIS1LRBA) | SPIS1RSTA;
     SPI1P = (1 << 19);
     SPI1CMD = SPIBUSY;
-
+    SPI1C2=(0x2<<SPIC2MOSIDN_S) | (0x1<<SPIC2MISODM_S); // GuaAck. 30.10.2018; timing of MISO 
```

Comment:
The PR already has the info of the code change author as well as the date and specific change diff, so putting that info in the comment isn't very useful. I suggest replacing the comment with more helpful info.
Example:
What do these registers mean? (Set Register blah, which controls/sets/manipulates bleh)
What do the operations do? (Set bit blah to do bleh)

Reply:
Explanation of the registers:
We couldn't find any documentation about SPI interface in 8266 but @GuaAck found out it is similar or same as in ESP32. I am particularly referring to "ESP32 Technical Reference Manual Version 3.1", chapter 7.8 (SPI) Registers, register SPI_CTRL2_REG. 
So we set the SPI_MISO_DELAY_MODE to 1 and on logic analyzer I saw the desired change.

Reply:
@JiriBilek thank you for the answer. Could you please update the PR and add it as a comment? I am aware that there is no documentation, so I am looking to explain these things directly in the code. The esp32 reference is also helpful.

Reply:
Can I update the PR, not being the OP?